lib/block-supports/layout.php

```diff
@@ -63,14 +63,18 @@ function gutenberg_render_layout_support_flag( $block_content, $block ) {
 	if ( $content_size || $wide_size ) {
 		ob_start();
 		?>
+			<?php echo '.wp-container-' . $id; ?> {
+				--wp--layout--content-size: <?php echo $content_size ? $content_size : 'var(--wp--layout--wide-size)'; ?>;
+				--wp--layout--wide-size: <?php echo $wide_size ? $wide_size : 'var(--wp--layout--content-size)'; ?>;
```

Comment:
why the fallback to 'var(--wp--layout--content-size)' ?

Reply:
It's basically the same behavior as what we previously had. 
For the content width we had `max-width: <?php echo $content_size ? $content_size : $wide_size; ?>;`
For wide we had `max-width: <?php echo $wide_size ? $wide_size : $content_size; ?>;`

We know that at least one of the 2 is defined (but not necessarily both). The cross-fallback will make it work correctly in all cases. The only thing that changes is that they will now be expressed as css-vars and will be available to other styles.

Reply:
I'm not sure I like the fallback to the CSS variables instead of just writing the value to the right variable 🤔 
It probably doesn't change anything though.

Reply:
btw, show we the same thing done here in `LayoutProvider` too (for the editor part)

Reply:
> I'm not sure I like the fallback to the CSS variables instead of just writing the value to the right variable 🤔
> It probably doesn't change anything though.

Done in https://github.com/WordPress/gutenberg/pull/30081/commits/45b732c4e5cc138b65c4509257282a618a3f8c63 👍 

> btw, show we the same thing done here in `LayoutProvider` too (for the editor part)

Done in https://github.com/WordPress/gutenberg/pull/30081/commits/2f299e2c9cfd1ac05d2616df62031c65cf4636da 👍